framework/include/cppmicroservices/LDAPFilter.h

```diff
@@ -130,10 +130,8 @@ class US_Framework_EXPORT LDAPFilter {
    *        in the match.
    * @return <code>true</code> if the bundle's properties match this
    *         <code>LDAPFilter</code> <code>false</code> otherwise.
-   * @throws std::runtime_error If the number of keys of the <code>bundle</code>
+   * @throws std::runtime_error If the number of keys of the bundle's properties
```

Comment:
We need to be careful about the terminology here. It is the bundle's manifest headers that are checked and matched. The return doc above also users the wrong properties term.